src/_guides/bump-sh-tutorials/documenting-multiple-api-versions.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Documenting Multiple API Versions
+authors: phil
+excerpt: When you release a new version of of an API you’re going to need to keep the old one around for a while, and keep the documentation up to date until it can be retired.
+date: 2025-05-22
+---
+
+
+Versioning an API can be incredibly difficult, but working out how to handle documentation for multiple APIs can be even more of a challenge. When you release a new version of of an API you’re going to need to keep the old one around for a while, and manage it as its own project until it can be retired. 
```

Comment:
```suggestion
Versioning an API can be incredibly difficult, but working out how to handle documentation for multiple APIs can be even more of a challenge. When you release a new version of an API you’re going to need to keep the old one around for a while, and manage it as its own project until it can be retired. 
```

Reply:
Would it make sense to make a nod to https://docs.bump.sh/guides/technical-writing/how-our-api-docs-can-ease-the-pain-of-api-deprecation/ ?